src/components/ChatWindow/ChatInput/ChatInput.tsx

```diff
@@ -81,9 +126,28 @@ export default function ChatInput({ conversation }: ChatInputProps) {
         value={messageBody}
       />
 
-      <button className={classes.sendButton} onClick={() => handleSendMessage(messageBody)} disabled={!isValidMessage}>
-        <SendMessageIcon />
-      </button>
+      <Grid container alignItems="flex-end" justify="flex-end" wrap="nowrap">
+        <input ref={fileInputRef} type="file" style={{ display: 'none' }} onChange={handleSendFile} />
```

Comment:
Can we filter the files here to only allow what Twilio Conversations support? https://www.twilio.com/docs/conversations/conversations-limits#media-limits
I believe you can use the `accept` attribute. Also, add check for max file size. If it's more than what Twilio Conversations support, don't even start uploading.

Reply:
I think those limits are only relevant to Conversations that send messages to an SMS channel. In Conversations that only have chat participants, any file type can be sent. All file types that I have uploaded in chat have succeeded.

Also, the Conversations SDK already checks the file size and lets us know when it is too large, in which case we display a useful error to the user:

![image](https://user-images.githubusercontent.com/12685223/112531103-cd94d680-8d6c-11eb-98a2-256131677c74.png)

Is there a benefit to checking the file size ourselves instead of letting the SDK do it?

Reply:
If the SDK is already checking it, then I think we're good. How about the file types? Is there a doc somewhere that mentions unsupported/supported files? For example, it's generally not advisable to upload executable files. I'm curious how the SDK or server handles this.

Reply:
It looks like there are no limits with respect to the file types that can be sent between two chat participants. I've even been able to send executable files in my tests.

So any file type limitations we add would be specific to our app only, and not specific to the Conversations platform. I'm happy to add some (it's technically easy), but I think this is a decision that Sean or Donal should weigh in on. I'll ask around.

Reply:
@charliesantos I added the `accept` attribute to the file input. I used some MIME types instead of file extensions just to allow for some greater flexibility. Also, using `image/*` allows mobile users to take a picture to upload to the chat. What do you think?

Reply:
It looks great! Thanks for adding this. 🚢  🚀